experiment/kind-multizone-e2e.sh

```diff
@@ -143,14 +143,35 @@ create_cluster() {
     ;;
   esac
 
-  # create the config file
+  # create the config file with 2 nodes per zone
   cat <<EOF > "${ARTIFACTS}/kind-config.yaml"
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 featureGates:
   TopologyAwareHints: true
 nodes:
 - role: control-plane
+- role: worker
+  kubeadmConfigPatches:
+  - |
+    kind: JoinConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "topology.kubernetes.io/zone=zone-a"
```

Comment:
Tiny non-blocking nit: This would be easier to read if the nodes were: AABBCC instead of the current ABCABC.

Reply:
done